requirements.txt

```diff
@@ -1 +1 @@
-libsigopt @ git+http://github.com/sigopt/libsigopt@c5c9d0d912718f36e20bbe9960aad802950e87c9
+libsigopt @ git+http://github.com/sigopt/libsigopt@19d481b58b7f18f7bcd4fdfff87da7c0eadf5a62
```

Comment:
not necessary just yet, but we should eventually use tags on libsigopt and reference this more cleanly